infra/modules/apim/policies/api_policy.xml

```diff
@@ -0,0 +1,181 @@
+<policies>
+    <inbound>
+        <base />
+        <!-- Getting the main variable where we keep the list of backends -->
+        <cache-lookup-value key="listBackends" variable-name="listBackends" />
+        <!-- If we can't find the variable, initialize it -->
+        <choose>
+            <when condition="@(context.Variables.ContainsKey("listBackends") == false)">
+                <set-variable name="listBackends" value="@{
+                    // -------------------------------------------------
+                    // ------- Explanation of backend properties -------
+                    // -------------------------------------------------
+                    // "url":          Your backend url
+                    // "priority":     Lower value means higher priority over other backends. 
+                    //                 If you have more one or more Priority 1 backends, they will always be used instead
```

Comment:
... have more [than] one /or more/ Priority 1 backends...

/or more/ should be removed